eng/build.sh

```diff
@@ -235,6 +240,14 @@ function BuildSolution {
     disable_parallel_restore=true
   fi
 
+  if [[ "$test_ioperation" == true ]]; then
+    export ROSLYN_TEST_IOPERATION="true"
+
+    if [[ "$test_mono" != true && "$test_core_clr" != true ]]; then
+      test_core_clr=true
```

Comment:
>test_core_clr=true [](start = 6, length = 18)

It doesn't look like any other option implies this.

Reply:
This mirrors `build.ps1` behavior.

---
In reply to: [430712569](https://github.com/dotnet/roslyn/pull/44436#discussion_r430712569) [](ancestors = 430712569)

Reply:
> This mirrors build.ps1 behavior.

Does this mean one cannot test IOperations for desktop?

---
In reply to: [430718240](https://github.com/dotnet/roslyn/pull/44436#discussion_r430718240) [](ancestors = 430718240,430712569)

Reply:
You can. On windows, this flag targets desktop by default, which is overridable. We can do that because the framework is installed on all platforms. For Mac/Linux, the default target framework is core clr, which is also overridable. We have no requirement that users install mono in order to do development on these platforms, and indeed I have not installed it. I did an initial smoke test of the IOperation flag on Linux, then went on my windows box to run the full set of tests (including IDE tests).

---
In reply to: [430719605](https://github.com/dotnet/roslyn/pull/44436#discussion_r430719605) [](ancestors = 430719605,430718240,430712569)